pkg/features/kube_features.go

```diff
@@ -649,6 +649,11 @@ const (
 	// Add support for the HPA to scale based on metrics from individual containers
 	// in target pods
 	HPAContainerMetrics featuregate.Feature = "HPAContainerMetrics"
+
+	// owner: TBD ?
```

Comment:
who is going to own it?

Reply:
That's why it's TDB :smile: it needs to be addressed :sweat_smile: Not sure how owning features gates works :)

Reply:
what about you? I mean, somebody has to take care to graduate it, usually the person that implements it, otherwise, if it is stale it eventually will be removed

Reply:
I mean I'm up for it, just not sure about the implications :smile:

Reply:
But yeah, if it's usually the person that implement's it, I'll do it

Reply:
Added mysel then :+1: